lib/task/import/validate/csvBaseValidator.class.php

```diff
@@ -54,27 +57,27 @@ public function __construct(?array $options = null)
         $this->testData = new CsvValidatorResult($this->title, $this->filename, $this->displayFilename, $this->getClassName());
     }
 
-    public function columnPresent(string $columnName, ?array $header = null)
+    public function columnPresent(string $columnName)
     {
         if (isset($this->columnPresent[$columnName])) {
             return $this->columnPresent[$columnName];
         }
 
-        if (!isset($this->columnPresent[$columnName]) && null !== $header) {
-            return $this->columnPresent[$columnName] = in_array($columnName, $header);
+        if (!isset($this->columnPresent[$columnName]) && null !== $this->header) {
```

Comment:
`isset($this->columnPresent[$columnName])` has already been tested in line 62, and must be false so it's not necessary to test it here.

Reply:
This is correct. I have updated this.